samples/openthread/cli/src/ble.c

```diff
@@ -15,6 +15,10 @@
 #define ADV_INT_MIN 0x01e0 /* 300 ms */
 #define ADV_INT_MAX 0x0260 /* 380 ms */
 
+#if defined(CONFIG_OPENTHREAD_BLE_TCAT)
+#error "Multiprotocol example and TCAT functionality do not coexist together."
```

Comment:
this error is thrown quite late into the building. How about adding to `CMakeLists.txt` : 
```
if (CONFIG_CLI_SAMPLE_MULTIPROTOCOL AND CONFIG_OPENTHREAD_BLE_TCAT)
        message( FATAL_ERROR "Multiprotocol example and TCAT functionality do not coexist together." )
endif()
```

Reply:
I'd suggest "Multiprotocol support and TCAT functionality are not compatible." to match the note that should be added to the readme.

Reply:
fixed.